powersimdata/data_access/tests/test_data_access.py

```diff
@@ -67,7 +67,7 @@ def _check_content(filepath):
         assert CONTENT == f.read()
 
 
-root_dir = server_setup.DATA_ROOT_DIR
+root_dir = server_setup.DATA_ROOT_DIR.rstrip("/")
```

Comment:
This is a funny artifact of the `config.ini` file I'm using with my other branch - it ends up overriding the `DATA_ROOT_DIR` with `LOCAL_DIR`, which is expected, but the trailing slash breaks a couple of tests here which use string concatenation to join paths for simplicity/readability. Real functionality uses `os.path.join` so this isn't a problem elsewhere.

Reply:
You lost me. In your other branch, `jon/windows` I presume, you have a `config.ini` file where you set the path for the various variables, including `DATA_ROOT_DIR`. But for what reason it is overwritten by `LOCAL_DIR`?

Reply:
The config.ini sets the deployment mode, which results in using a `LocalConfig`, where we default the `DATA_ROOT_DIR` to be equal to `LOCAL_DIR` since there is no need for separate locations. This caused a test failure since `LOCAL_DIR` has an extra `/` at the end. So this is just to make the tests work whether or not there is a config.ini causing this effect.